### ISSUE TITLE
QuickOpen: add aria-setsize and aria-posinset

### ISSUE THREAD

**Issue by bhagwatshelke1047** [microsoft/vscode#76709]
**Environment Details:**
VSCode Version : 1.36.1 (user setup)
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
Screen Reader Versions : JAWS 2019

**Repro Steps:**
1.Open Visual studio code editor.
2.Open the command Pallet by pressing "F1" key and type "Theme"
3.Using Down arrow key select "Preference colour Theme" and press enter key to select.
4.Using "Up/Down" arrow key select different available theme.
5.Start screen reader.

**Actual:**
 Screen reader does not announce the ordering that is 'light 1 of 15' for drop down item of 'select colour theme' combo box.

This issue is observed in Win10 + JAWS

**Expected:**
Screen reader should announce the proper ordering that is 'light 1 of 15' for each drop down item of 'select colour theme' combo box.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
  Screen reader users will not get the exact information.

**MAS Reference:**
**MAS 1.3.1 -** https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_VSCode_ChangingoftheTheme_ScreenReader.pptx](https://github.com/microsoft/vscode/files/3362387/A11y_VSCode_ChangingoftheTheme_ScreenReader.pptx)

**Comment by aeschli** [comment]
@isidorn I'm using the quick pick service for that and I think this should be added there (or we need to add API so that I can set the labels that are read out)

**Comment by isidorn** [comment]
Currently we do not support JAWS, only NVDA on windows and VoiceOver on mac. More about this can be found here https://code.visualstudio.com/docs/editor/accessibility#_screen-readers

And this works just fine with VoiceOver thus closing.
Please let me know if you can reproduce this with NVDA and I will reopen

![lala](https://user-images.githubusercontent.com/1926584/60803285-24aede00-a17b-11e9-8570-c269d61d9cc5.gif)

**Comment by isidorn** [comment]
However I do agree we should read the order. Due to that reopening (since I originally misunderstood the issue).
For this we need to add `aria-setsize` and `aria-posinset` attributes.

@bpasero can you provide some code pointers since @chrmarti is out? So I try to add these attributes?

**Comment by bpasero** [comment]
@isidorn this is not quick open related is it? these attributes should be on the tree/list layer, no?

**Comment by isidorn** [comment]
They are already on the list layer. But the quick open is using the old tree.
Thus duplicate of https://github.com/microsoft/vscode/issues/69955

**Comment by bpasero** [comment]
@isidorn not sure, all pickers use the new tree and I think this is about the theme picker?

**Comment by isidorn** [comment]
@bpasero you are correct. We are using the new tree and the `aria-setsize` and `aria-posinset` are set.
This all works fine on NVDA and voice over. The issue here is that the tester is using JAWS, which we do not support. More details about this can be found here (we support NVDA) https://code.visualstudio.com/docs/editor/accessibility. 

<img width="1154" alt="Screenshot 2019-07-08 at 14 08 02" src="https://user-images.githubusercontent.com/1926584/60809000-e10fa080-a189-11e9-8c61-917483ef4f7d.png">
